data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md

```diff
@@ -6,7 +6,5 @@
      1. In the left sidebar, click **Enterprise overview**. 
      1. {% endif %} In the enterprise sidebar, {% octicon "law" aria-label="The law icon" %} **Policies**.
 1. Navigate to the {% data variables.product.prodname_actions %} settings:
-   * **In an organization or repository**: click **Actions** in the left sidebar.
-     
-     ![Actions setting](/assets/images/help/settings/settings-sidebar-actions.png)
-   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**{% endif %}: click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
+   * **In an organization or repository**: click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then **Runners**{% endif %}.
+   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
```

Comment:
This was missing in [this procedures](https://docs-6161--patch-1.herokuapp.com/en/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners#creating-a-custom-label) but you should probably check it's not creating duplicate steps in any other procedures.

```suggestion
   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies", then click the **Runners** tab.
```